docs/routes.md

```diff
@@ -0,0 +1,268 @@
+
+
+### Remove a scheduled item
+
+`DELETE /schedule/:some-id` will remove the scheduling of a publish in the future.  On success, it will return a 200 with
+```json
+{
+  "at": 44392893402093,
+  "publish": "<uri that was to be published>"
+}
+```
+
+## Versions
+
+### Propagating Versions
+
+Some versions have a special behavior called _propagating_, which any reference within their data is changed to be the same version as itself.
+
+The current propagating versions are:
+* published
+* latest
+
+## RESTful API
+
+We try to follow the [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) standard when making decisions about changes to the API.  In particular, we are adhering to
+
+1.  _Individual resources are identified by URI._  Therefore, there is no need to return an id as part of the data of a request if that id was used to fetch the data.  In many environments (production, development, staging) hostnames can be dynamic, so we can optionally omit the hostname and assume that the resource is available at the same host as a referring resource.
+
+2.  _When a client holds a representation of a resource, it has enough information to modify the resource._  Any random ordering of `GET` and `PUT` requests to the same URI with the returned data will not result in any change of state (beyond the separate audit trail that such operations took place).
+
+3.  _Messages always include enough information to process a resource, including MIME type and cacheability._  All resources are returned with appropriate Content-Type, Cache, and Vary headers.  That means resources that should be cached, like published content, are marked appropriately as cacheable.
+
+4.  _HATEOAS: no assumptions about additional actions are made without being described in the resource itself._  Excluding the basic HTTP method calls, we tell the client about additional resources available related to the current resource.  
+
+    Currently, we link to child resources with `{ _ref: 'domain.com/full/resource/uri' }`. There is no assumption that the client knows about these child resources implicitly.  More importantly, we don't expect the client to know about where resources are, their types, or that they should be linked together in any particular way, so we always reference to linked resources with a full URI.
+
+    For more information about the design decisions of `_ref`, see https://github.com/nymag/amphora/issues/108
+
+### Namespacing
+
+It might be a good idea to namespace this API behind versions, such as `/v1/` or `/v2/`, or `/services/`, or even `/clay/`.  This can be done by adjusting the hostname, or through a reverse proxy like Varnish.  Putting different versions behind Varnish might be particularly useful so that previous versions of an application can be run at the same time as newer versions to service old and new clients until previous versions can be deprecated.
```

Comment:
```suggestion
It might be a good idea to namespace this API behind versions, such as `/v1/` or `/v2/`, or `/services/`, or even `/clay/`. This can be done by adjusting the hostname, or through a reverse proxy like Varnish. Putting different versions behind Varnish might be particularly useful so that previous versions of an application can be run at the same time as newer versions to service old and new clients until previous versions can be deprecated.
```